### PR TITLE
[terra-date-picker] Add `value` prop in DatePickerField to align with DatePicker.

### LINKED ISSUE #1152: [terra-date-picker] Missing `value` prop in DatePickerField
# Bug Report

## Description
- DatePickerField does not expose the `value` prop that is in DatePicker to allow a controlled DatePickerField.
- The DatePickerFIeld [example](https://engineering.cerner.com/terra-ui/components/terra-date-picker/date-picker/date-picker-field) renders a DatePicker instead of a DatePickerField.
- The DisabledDatePickerFIeld test throws a javascript warning for not setting the required `datePickerId` prop.

## Additional Context / Screenshots
```
checkPropTypes.js?a15c:20 Warning: Failed prop type: The prop `datePickerId` is marked as required in `DatePickerField`, but its value is `undefined`.
    in DatePickerField (created by DatePickerDisabledDate)
    in DatePickerDisabledDate (created by CodesplitWrapper)
```

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

### PR DESCRIPTION
**Pull request by benbcai**
### Summary
- Added `value` prop in DatePickerField to align with DatePicker.
- Fixed import in DatePickerFIeld example.
- Fixed javascript warning in DisabledDatePickerFIeld test.

Closes #1152 

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-date-picker/src/terra-dev-site/doc/example/DatePickerField.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
-import DatePickerField from 'terra-date-picker';
+import DatePickerField from '../../../DatePickerField';
```
Comment (supreethmr) [inline packages/terra-date-picker/src/terra-dev-site/doc/example/DatePickerField.jsx:4]:
```suggestion
import DatePickerField from 'terra-date-picker/lib/DatePickerField';
```
nit : I think we have decided to not have import with file-path on examples page.

**Comment by mjhenkes** [comment]
@benbcai I bet the screenshot failure was cause by the terra-fusion line height theme change here being slightly off: https://github.com/cerner/terra-core/pull/3036#event-3440549459

We may want to get this in, and get another release of terra-core before we update the screenshot.

**Comment by benbcai** [comment]
> @benbcai I bet the screenshot failure was cause by the terra-fusion line height theme change here being slightly off: [cerner/terra-core#3036 (comment)](https://github.com/cerner/terra-core/pull/3036#event-3440549459)
> 
> We may want to get this in, and get another release of terra-core before we update the screenshot.

@mjhenkes Yes, that was the cause. I was thinking of updating the screenshots for now to get framework building again and then revert once terra-core is released on Tuesday. But I can hold of on this and just wait for terra-core's release.

**Comment by mjhenkes** [comment]
@benbcai I'll get a release out for you today so you don't have to wait.

**Comment by benbcai** [comment]
Thanks @mjhenkes !

**Pull request merged**